guides/getting-started.md

```diff
@@ -43,8 +43,8 @@ To use the pre-configured component generators you have to create your generator
 import ReactGenerator from '@teleporthq/teleport-component-generator-react'
 
 const uidl = { ... } // your sample here
-
-const { files } = await ReactGenerator.generateComponent(uidl)
+const generator = ReactGenerator.createReactComponentGenerator();
```

Comment:
createReactComponentGenerator should be imported directly from '@teleporthq/teleport-component-generator-react' and used like that, without the ReactGenerator prefix